blackbox_repository/conversion_scripts/scripts/nasbench201_import.py

```diff
@@ -0,0 +1,212 @@
+import bz2
+import pickle
+
+import matplotlib.pyplot as plt
+
+import pandas as pd
+import numpy as np
+from blackbox_repository.blackbox_tabular import serialize, BlackboxTabular
+import sagemaker_tune.search_space as sp
+from blackbox_repository.conversion_scripts.utils import repository_path
+from syne_tune.util import catchtime
+
+
+def str_to_list(arch_str):
+
+    node_strs = arch_str.split('+')
+    config = []
+    for i, node_str in enumerate(node_strs):
+        inputs = list(filter(lambda x: x != '', node_str.split('|')))
```

Comment:
list comprehension are generally recommended over list + filter:
```suggestion
        inputs = [x for x in node_str.split('|') if x != '']
```